packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -227,8 +227,8 @@ class OAuthClient extends CozyStackClient {
       softwareID: this.oauthOptions.softwareID
     }
     const data = this.snakeCaseOAuthData({
-      ...mandatoryFields,
-      ...information
+      ...information,
+      ...mandatoryFields
```

Comment:
in fact @zatteo I think we misunderstood what's happening. This is not fields that we can't override, this is juste mandatoryFields. So we can override them if wanted, but they need to be there. 

So I think we can close the issue

Reply:
Oh you're right. I close.